src/site/content/en/blog/defining-core-web-vitals-thresholds/index.md

```diff
@@ -0,0 +1,486 @@
+
+[Core Web Vitals](/vitals/#core-web-vitals) are a set of field metrics that
+measure important aspects of real-world user experience on the web. Core Web
+Vitals includes metrics, as well as target thresholds for each metric, which
+help developers qualitatively understand whether the experience of their site is
+good, needs improvement, or is poor. This post will explain the approach used to
+choose thresholds for Core Web Vitals metrics generally, as well as how the
+thresholds for each specific Core Web Vitals metric were chosen.
+
+## Refresher: Core Web Vitals metrics and thresholds
+
+In 2020 the Core Web Vitals are three metrics: Largest Contentful Paint (LCP),
+First Input Delay (FID), and Cumulative Layout Shift (CLS). Each metric measures
+a different aspect of user experience: LCP measures perceived load speed and
+marks the point in the page load timeline when the page's main content has
+likely loaded; FID measures responsiveness and quantifies the experience users
+feel when trying to first interact with the page; and CLS measures visual
+stability and quantifies the amount of unexpected layout shift of visible page
+content.
+
+Each Core Web Vitals metric has associated thresholds, which categorize
+performance as either "good", "needs improvement", or "poor":
+
+<div class="w-stack w-stack--center w-stack--md">
+  <img src="/vitals/lcp_ux.svg" width="400px" height="350px"
+       alt="Largest Contentful Paint threshold recommendations">
+  <img src="/vitals/fid_ux.svg" width="400px" height="350px"
+       alt="First Input Delay threshold recommendations">
+  <img src="/vitals/cls_ux.svg" width="400px" height="350px"
+       alt="Cumulative Layout Shift threshold recommendations">
+</div>
+
+<div class="w-table-wrapper">
+  <table>
+    <tr>
+      <th>&nbsp;</th>
+      <th>Good</th>
+      <th>Poor</th>
+      <th>Percentile</th>
+    </tr>
+    <tr>
+      <td>Largest Contentful Paint</td>
+      <td>≤2500ms</td>
+      <td>>4000ms</td>
+      <td>75</td>
+    </tr>
+    <tr>
+      <td>First Input Delay</td>
+      <td>≤100ms</td>
+      <td>>300ms</td>
+      <td>75</td>
+    </tr>
+    <tr>
+      <td> Cumulative Layout Shift</td>
+      <td>≤0.1</td>
+      <td>>0.25</td>
+      <td>75</td>
+    </tr>
+</table>
+
+Additionally, to classify the overall performance of a page or site, we use the
+75th percentile value of all page views to that page or site. In other words, if
+at least 75 percent of page views to a site meet the "good" threshold, the site
+is classified as having "good" performance for that metric. Conversely, if at
+least 25 percent of page views meet the "poor" threshold, the site is classified
+as having "poor" performance. So, for example, a 75th percentile LCP of 2
+seconds is classified as "good", while a 75th percentile LCP of 5 seconds is
+classified as "poor".
+
+## Criteria for the Core Web Vitals metric thresholds
+
+When establishing thresholds for Core Web Vitals metrics, we identified criteria
```

Comment:
```suggestion
When establishing thresholds for Core Web Vitals metrics, Google identified criteria
```
Let's explicitly identify who "we" is here (it's actually the second instance of the use of "we", but this is a good place to define it)